modules/create.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <title>Paper Mechatronics - Create</title>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.12.2/jquery.min.js"></script>
+    <link rel="shortcut icon" type="image/png" href="img/logo_webTab.png" />
+    <link
```

Comment:
Is it safe to mix and match bootstrap versions here?

Reply:
Sorry i forgot to change the import of bootstraps js in the create module.